feature/feature.go

```diff
@@ -54,7 +60,8 @@ func init() {
 // To add a new feature, define a key for it above and add it here.
 var defaultClusterAPIFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	// Every feature should be initiated here:
-	MachinePool:        {Default: false, PreRelease: featuregate.Alpha},
-	ClusterResourceSet: {Default: true, PreRelease: featuregate.Beta},
-	ClusterTopology:    {Default: false, PreRelease: featuregate.Alpha},
+	MachinePool:                    {Default: false, PreRelease: featuregate.Alpha},
+	ClusterResourceSet:             {Default: true, PreRelease: featuregate.Beta},
+	ClusterTopology:                {Default: false, PreRelease: featuregate.Alpha},
+	KubeadmBootstrapFormatIgnition: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
Does the feature gate affect the API changes? is it independent? cc @vincepri @CecileRobertMichon @fabriziopandini

Reply:
The *general* policy, as far as I could gather from discussions around released ClusterClass structs in the Cluster CRD, is that even though the feature gate gates the API fields those fields still have the same guarantees as every other field in those (v1beta1) resources.

Reply:
I'll mark it as resolved as @enxebre ACK'ed to keep the remaining discussions focused.